### ISSUE TITLE
Request : Add file path and line number to make MarkedPlugin error messages more friendly.

### ISSUE THREAD

**Issue by MasatoMakino** [TypeStrong/typedoc#1557]
## Search Terms

error message, getHighlighted, markedOptions, highlight

## Problem

Currently, the error messages of the `getHighlighted` function of `MarkedPlugin.ts` do not show the file path and line number of the file where the error occurred. So users need a lot of time to identify the cause of the error message.

## Suggested Solution

Currently, `getHighlighted` generates the following error messages

https://github.com/TypeStrong/typedoc/blob/315bd2501360c9e3738c512a850f37cf314b8627/src/lib/output/plugins/MarkedPlugin.ts#L109-L121

```
Unsupported highlight language "{language type}" will not be highlighted. Run typedoc --help for a list of supported languages.
```

The following information will be added to this message.

```
This code block is in line n of "path/to/file".
```

**Comment by Gerrit0** [comment]
This is a good idea 👍

**Comment by Gerrit0** [comment]
Resolved by #1561
